src/complete.ts

```diff
@@ -4,6 +4,10 @@ import {syntaxTree} from "@codemirror/language"
 import {SyntaxNode} from "@lezer/common"
 import {Type, Keyword} from "./sql.grammar.terms"
 
+function isEmptyStringNode(doc: Text, node: SyntaxNode) {
+  return doc.sliceString(node.from, node.to).trim() === '';
+}
```

Comment:
The name is misleading - this detects whitespace nodes, e.g. `'  '` will be caught.